util/tracing/trace_to_csv.c

```diff
@@ -55,13 +55,15 @@ int table_size;
  * Print a usage message.
  */
 void usage() {
-    printf("\nUsage: trace_to_csv [options] trace_file_root (without .lft extension)\n\n");
-    /* No options yet:
+    printf("\nUsage: trace_to_csv [options] trace_file_root (with .lft extension)\n\n");
     printf("\nOptions: \n\n");
-    printf("  -f, --fast [true | false]\n");
-    printf("   Whether to wait for physical time to match logical time.\n\n");
+    printf("  -s, --start [time_spec] [units]\n");
+    printf("   The target time to begin tracing.\n\n");
+    printf("  -e, --end [time_spec] [units]\n");
+    printf("   The target time to stop tracing.\n\n");
+    // printf("  -f, --fast [true | false]\n");
+    // printf("   Whether to wait for physical time to match logical time.\n\n");
```

Comment:
Is there any reason why '-f' option is commented out? If you're going to support this later, maybe mark it as a TODO or FIXME?

Reply:
I just inherited the original code below. However, the comment does not look like relating to the visualization tool. I simply removed this part in [6ce0e10](https://github.com/lf-lang/reactor-c/pull/320/commits/6ce0e103dd6ce77ec3e7e3645525c68cd70f5a36).  https://github.com/lf-lang/reactor-c/blob/1f97e6a9eb5dd7c82fb1c1bd226ed66283d05137/util/tracing/trace_to_csv.c#L58-L64